packages/framework/src/hooks/useMockResponse.ts

```diff
@@ -0,0 +1,5 @@
+export let useMockResponse: boolean = false;
```

Comment:
Like we discussed this should end up being a map of appIds -> config object (that extends the app model) with the useMockResponse attribute in it. Then you can pass the object around and use assignment on it.

Think of this map as a global store of app configuration.